packages/nimble-components/src/combobox/testing/combobox.pageobject.ts

```diff
@@ -0,0 +1,128 @@
+import { keyEnter, keyArrowDown } from '@microsoft/fast-web-utilities';
+import type { Combobox } from '..';
+import { waitForUpdatesAsync } from '../../testing/async-helpers';
+import {
+    createEventListener,
+    waitAnimationFrame
+} from '../../utilities/tests/component';
+
+/**
+ * Page object for the `nimble-combobox` component to provide consistent ways
+ * of querying and interacting with the component during tests.
+ */
+export class ComboboxPageObject {
```

Comment:
> I don't see a getOptions() method here so it'd be good to add and ensure it has a test exercising it.

I didn't add that since `filteredOptions` is part of the combobox's public API, unless we forgot to mark it internal. The select has the same property, but it is internal.

> 2. get an array of all option values as string

Not sure if that's necessary if `filteredOptions` is part of the public API (e.g. `combobox.filteredOptions.map(x => x.value)`), but I guess there's no harm in adding it for convenience.

Reply:
I wouldn't expect filteredOptions to be part of the public API. We should mark that internal.